cmd/containerd-shim-runhcs-v1/log.go

```diff
@@ -0,0 +1,54 @@
+/*
+// TODO: Move this to docs and don't merge
+
+Example logger, which writes to 2 different files
+*/
+
+package main
+
+import (
+	"context"
+	"io"
+	"os"
+	"sync"
+
+	"github.com/Microsoft/hcsshim/cmd/logging"
+)
+
+func _main() {
+	logging.Run(logger)
+}
+
+func logger(_ context.Context, config *logging.Config, ready func() error) error {
+	var wg sync.WaitGroup
+	wg.Add(2)
+
+	fileOut, err := os.Create("container-stdout.txt")
+	defer fileOut.Close()
```

Comment:
Move this below the error check, otherwise if the file creation fails this will panic as fileOut will be nil and we'll be trying to invoke a method on it.